libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/infopanel/InfoPanelCoordinator.kt

```diff
@@ -81,6 +81,14 @@ internal class InfoPanelCoordinator(
         coroutineScope.launch {
             context.systemBarsInsets.collect { updateGuidelinePosition(systemBarsInsets = it) }
         }
+        coroutineScope.launch {
+            context.styles.infoPanelGuidelineMaxPosPercent.collect {
```

Comment:
Don't you want to test it?

Reply:
I tried, but unfortunately, Robolectric was throwing exceptions when I tried to layout the view (`parentLayout.layout(0, 0, 200, 300)`) to get concrete measurements in tests. After fighting with this for over 4 hrs I had to throw the towel.